whitepapers/0010_ibc_generic_messaging

```diff
@@ -0,0 +1,92 @@
+# IBC Bridging
+
+## Objective
+
+Since Wormchain is a cosmos-sdk based chain that is IBC-enabled, we can leverage IBC generic messaging to reduce the operational burden required to run a guardian node. Wormhole guardians should, therefore, be capable of scaling to support all IBC-enabled chains at minimal cost while only running a full node for Wormchain.
+
+## Background
+
+[IBC](https://ibcprotocol.org/) is the canonical method of generic message passing within the cosmos ecosystem. IBC is part of the cosmos-sdk and Cosmos chains can enable it to connect with other cosmos chains.
+
+[Wormchain](https://github.com/wormhole-foundation/wormhole/tree/main/wormchain) is a cosmos-sdk based blockchain that has been purpose-built to support Wormhole. It allows the guardians to store global state on all the blockchains which wormhole connects to, and enables a suite of product and security features.
+
+## Goals
+
+- Remove the requirement for guardians to run full nodes for IBC-enabled chains. Guardians should be able to support any IBC-enabled chain by only running a full wormchain node.
+- Define a custom IBC specification for passing wormhole messages between IBC chains and wormchain.
+- Ensure this design is backwards-compatible with existing Cosmos integrations.
+- Ensure this design does not violate any of Wormhole's existing security assumptions.
+
+## Non-Goals
+
+This document does not propose new cosmos networks for Wormhole to support. It is focused on the technical design of using IBC generic messaging to reduce the operational load on wormhole guardians.
+
+This document is also not meant to describe how wormhole can be scaled beyond the cosmos ecosystem.
+
+## Overview
+
+Currently wormhole guardians run full nodes for every chain that wormhole is connected to. This is done for maximum security and decentralization. Since each guardians runs a full node for each chain, they are able to independently verify the authenticity of wormhole messages that are posted on different blockchains. However, running full nodes has its drawbacks. Specifically, adding new chains to wormhole has a high operational cost per chain, which makes it difficult to scale wormhole.
```

Comment:
```suggestion
Currently, wormhole guardians run full nodes for every chain that wormhole is connected to. This is done to maximize security and decentralization. Since each guardian runs a full node for each chain, they are able to independently verify the authenticity of wormhole messages posted on different blockchains. However, running full nodes has its drawbacks. Specifically, adding new chains to wormhole has a high operational cost per chain, which makes it difficult to scale wormhole.
```

I removed a bit of passive voice.